packages/requisitions/src/RequestRequisition/DetailView/Toolbar/Toolbar.tsx

```diff
@@ -69,11 +63,15 @@ export const Toolbar: FC = () => {
             </Box>
           </Box>
         </Grid>
-        <DropdownMenu disabled={isDisabled} label={t('label.select')}>
-          <DropdownMenuItem IconComponent={DeleteIcon} onClick={onDelete}>
-            {t('button.delete-lines', { ns: 'distribution' })}
-          </DropdownMenuItem>
-        </DropdownMenu>
+        <Box
+          flexDirection="column"
+          alignItems="flex-end"
+          display="flex"
+          gap={2}
+        >
+          <ToolbarActions />
+          <ToolbarDropDown />
```

Comment:
took the drop down out and also the actions. I imagine the drop down will someday get more actions added.. and more actions in the actions component